src/showcase/rollup.config.js

```diff
@@ -1,20 +1,24 @@
 const node = require('rollup-plugin-node-resolve');
+const json = require('rollup-plugin-json');
 const commonjs = require('rollup-plugin-commonjs');
 const alias = require('@rollup/plugin-alias');
 
 module.exports = {
   plugins: [
     alias({
       // Work-around for https://github.com/bazelbuild/rules_nodejs/issues/1380#issuecomment-558631283
-      entries: ['business', 'core', 'icons', 'keycloak', 'maps', 'public'].map((pkg) => ({
-        find: `@sbb-esta/angular-${pkg}`,
-        replacement: `${__dirname}/../angular-${pkg}`,
-      })),
+      entries: ['business', 'core', 'icons', 'keycloak', 'maps', 'maps-leaflet', 'public'].map(
+        pkg => ({
+          find: `@sbb-esta/angular-${pkg}`,
+          replacement: `${__dirname}/../angular-${pkg}`
+        })
+      )
     }),
     node({
       mainFields: ['es2015', 'module', 'browser', 'jsnext:main', 'main'],
-      extensions: ['.mjs'],
+      extensions: ['.mjs']
     }),
     commonjs(),
-  ],
+    json()
```

Comment:
Why is json necessary?

Reply:
When I first ran `yarn build:showcase` i ran into an issue, that it could not read the esri-leaflet plugin until I add the json plugin.
Have a look at the error message:
`[!] Error: Unexpected token (Note that you need rollup-plugin-json to import JSON files)
node_modules/esri-leaflet/package.json (2:8)`

is there another way on how I can get this working?

Reply:
No, it is definitely required if esri-leaflet internally imports json. Could you add a comment that esri-leaflet requires the json plugin?

Reply:
Esri-leaflet is only used in the showcase to add some ArcGIS (ESRI) services on the map. I could also remove it and use different data, if you think it blows the showcase up too much. 
I just used `esri-leaflet` to show how easily one can integrate ArcGIS services in a leaflet map. Otherwise I will add the comment.

Reply:
I think it provides a good example on how to integrate the service we use at SBB. So I would say don't remove esri-leaflet, but add a comment in the rollup.config.js to indicate why we need the json plugin.